client/python/cryoet_data_portal/src/cryoet_data_portal/_models.py

```diff
@@ -253,58 +246,58 @@ def download_everything(self, dest_path: Optional[str] = None):
-        reconstruction_software (str):Name of software used for reconstruction
-        s3_mrc_scale0 (str):S3 path to this tomogram in MRC format (no scaling)
-        s3_omezarr_dir (str):S3 path to this tomogram in multiscale OME-Zarr format
-        scale0_dimensions (str):comma separated x,y,z dimensions of the unscaled tomogram
-        scale1_dimensions (str):comma separated x,y,z dimensions of the scale1 tomogram
-        scale2_dimensions (str):comma separated x,y,z dimensions of the scale2 tomogram
+        reconstruction_method (str): Describe reconstruction method (Weighted backprojection, SART, SIRT)
+        reconstruction_software (str): Name of software used for reconstruction
+        s3_mrc_scale0 (str): S3 path to this tomogram in MRC format (no scaling)
+        s3_omezarr_dir (str): S3 path to the this multiscale omezarr tomogram
+        scale0_dimensions (str): comma separated x,y,z dimensions of the unscaled tomogram
+        scale1_dimensions (str): comma separated x,y,z dimensions of the scale1 tomogram
+        scale2_dimensions (str): comma separated x,y,z dimensions of the scale2 tomogram
         size_x (int): Number of pixels in the 3D data fast axis
         size_y (int): Number of pixels in the 3D data medium axis
-        size_z (int):  Number of pixels in the 3D data slow axis.  This is the image projection direction at zero stage tilt
+        size_z (int): Number of pixels in the 3D data slow axis.  This is the image projection direction at zero stage tilt
         tomogram_version (str): Version of tomogram using the same software and post-processing. Version of tomogram using the same software and post-processing. This will be presented as the latest version
-        tomogram_voxel_spacing (TomogramVoxelSpacing): An object relationship with a specific voxel spacing for this experiment run
-        type (str):  Tomogram purpose (ex: CANONICAL)
+        tomogram_voxel_spacing (TomogramVoxelSpacing): An object relationship
+        tomogram_voxel_spacing_id (int): None
         voxel_spacing (float): Voxel spacing equal in all three axes in angstroms
     """
 
-    _gql_type = "tomograms"
+    _gql_type: str = "tomograms"
 
-    tomogram_voxel_spacing: TomogramVoxelSpacing = ItemRelationship(
-        TomogramVoxelSpacing,
-        "tomogram_voxel_spacing_id",
+    affine_transformation_matrix: str = StringField()
+    authors: List[TomogramAuthor] = ListRelationship(
+        "TomogramAuthor",
         "id",
+        "tomogram_id",
     )
-
-    id: int = IntField()
-    affine_transformation_matrix: str = StringField()
-    ctf_corrected: int = BooleanField()
+    ctf_corrected: bool = BooleanField()
     deposition_id: int = IntField()
     fiducial_alignment_status: str = StringField()
     https_mrc_scale0: str = StringField()
     https_omezarr_dir: str = StringField()
-    is_canonical: int = BooleanField()
```

Comment:
I assume this was a copy/paste error and that the attribute annotation should be `bool` (and not `int`) for `BooleanField`.

Reply:
Yep almost certainly a typo!